### PR TITLE
SAP Document Management Service Plugin Documentation

### PR DESCRIPTION
**Pull request by rashmiangadi11**
Adding the documentation for SAP Document Management Service plugin

### CODE REVIEW

File: plugins/index.md [hunk 1]
```diff
@@ -303,6 +303,40 @@ Available for:
...
+
+```cds
+extend my.Incidents with {
+  attachments: Composition of many Attachments
+}
+```
+
+![Screenshot showing the Attachments Table in a fiori app](assets/index/sdm-table.png)
+
+It also provides a CAP-level, easy-to-use integration of the [SAP Document Management Service](https://help.sap.com/docs/document-management-service).
```
Comment (smahati) [inline plugins/index.md:321]:
```suggestion
```

File: plugins/index.md [hunk 2]
```diff
@@ -303,6 +303,40 @@ Available for:
...
+
+Features:
+
+- Pre-defined type `Attachment` to use in entity definitions
+- Automatic handling of all upload,download,delete and rename operations
```
Comment (smahati) [inline plugins/index.md:326]:
```suggestion
- Automatic handling of all operations including upload, download, delete, and rename
```

File: plugins/index.md [hunk 3]
```diff
@@ -303,6 +303,40 @@ Available for:
...
+- SAP Document Management Service has inbuilt virus scanning feature.
```
Comment (smahati) [inline plugins/index.md:327]:
```suggestion
- SAP Document Management Service has a built in virus scanning feature.
```

File: plugins/index.md [hunk 4]
```diff
@@ -303,6 +303,40 @@ Available for:
...
+- (Automatic) Fiori Annotations for Upload Controls
+- Streaming and piping to avoid memory overloads
+
+Outlook:
+
+- Multitenancy intrinsically handled by the plugin
```
Comment (smahati) [inline plugins/index.md:333]:
```suggestion
- Intrinsic multitenancy handling
```

File: plugins/index.md [hunk 5]
```diff
@@ -303,6 +303,40 @@ Available for:
...
+
+Features:
+
+- Pre-defined type `Attachment` to use in entity definitions
+- Automatic handling of all upload,download,delete and rename operations
+- SAP Document Management Service has inbuilt virus scanning feature.
+- (Automatic) Fiori Annotations for Upload Controls
+- Streaming and piping to avoid memory overloads
+
+Outlook:
```
Comment (smahati) [inline plugins/index.md:331]:
```suggestion
Future enhancements:
```

**Review by smahati — CHANGES_REQUESTED** [review]

File: plugins/index.md (hunk 5 above)
Comment (rashmiangadi11) [reply plugins/index.md]: is it mandatory to add enhancements or can be skipped

**Review by smahati — APPROVED** [review]

**Pull request merged**